examples/bar.jl

```diff
@@ -35,38 +34,32 @@ function bar4()
     data = bar(;x = ["Product A", "Product B", "Product C"],
                 y = [20, 14, 23],
                 text = ["27% market share","24% market share", "19% market share"],
-                marker_color = "rgb(158,202,225)",
-                marker_opacity = 0.6,
-                line_color = "rgb(8,48,107)",
-                line_width = 1.5)
+				marker = attr(color = "rgb(158,202,225)", opacity = 0.6),
```

Comment:
Sounds good. I have a feeling it might have something to do with softtabstop. It works differently depending on its value comp. to tabstop and whether or not expandtab is set, so I'll see if working without it fixes the spacing issue. Also @spencerlyon2  are there any other fixes that need to be made?